### PR TITLE
Some fixes to "maintain-guides-async-backing.md"

### PR DESCRIPTION
**Pull request by mrcnski**
Addresses some issues found in https://github.com/paritytech/polkadot-sdk/pull/2826.

### CODE REVIEW

File: docs/maintain/maintain-guides-async-backing.md [hunk 1]
```diff
@@ -125,6 +126,8 @@ This phase involves configuring your parachain’s runtime to make use of async
 
      ![cargo-toml](../assets/async/async-backing-cargo.png)
 
+   - In the same file, add `"cumulus-primitives-aura/std",` to the `std` feature.
```
Comment (joepetrowski) [inline docs/maintain/maintain-guides-async-backing.md:129]: In L122 above, can you preformat `AuraUnincludedSegmentApi`?

**Review by joepetrowski — APPROVED** [review]

**Review by DrW3RK — APPROVED** [review]
Thank you!

**Comment by evilrobot-01** [comment]
Seems that this PR broke the screenshot image for implementing the `AuraUnincludedSegmentApi` within a runtime (see Phase 1, Step 7, substep 3 at https://wiki.polkadot.network/docs/maintain-guides-async-backing#phase-1---update-parachain-runtime) due to using the wrong image name.

The replaced image aligned with the config at https://github.com/paritytech/polkadot-sdk/blob/c01dbebeaa6394691974de46dd2d41a582f6a4c2/cumulus/parachains/runtimes/testing/rococo-parachain/src/lib.rs#L839

Edit: note also that the mention of async backing already being part of the cumulus parachain template is incorrect. The link points to a branch on the old cumulus repo and the cumulus template within the Polkadot SDK repo does not yet seem to be async backing ready.

**Comment by mrcnski** [comment]
You're right, should have created a new image called async-backing-unincluded-segment-**capacity**.png. Sorry, I missed that the same path was used in two places. Will fix it soon.

> Edit: note also that the mention of async backing already being part of the cumulus parachain template is incorrect.

Where is this mention? Let me know what and how to fix and I'll include it in the follow-up PR.

**Comment by evilrobot-01** [comment]
Hmmm, it seems there is indeed some async backing stuff in https://github.com/paritytech/polkadot-sdk/tree/master/cumulus/parachain-template, so I guess https://github.com/mrcnski/polkadot-wiki/blob/b24edec5c35be36afa772f97ccb9a1530d21812c/docs/maintain/maintain-guides-async-backing.md?plain=1#L16C3-L16C3 just needs updating to point to that location.

**Pull request merged**